app/src/main/java/org/cru/godtools/activity/BasePlatformActivity.kt

```diff
@@ -250,6 +251,7 @@ abstract class BasePlatformActivity : BaseDesignActivity(), NavigationView.OnNav
                 loginItem = findItem(R.id.action_login)
                 signupItem = findItem(R.id.action_signup)
                 logoutItem = findItem(R.id.action_logout)
+                tutorialItem = findItem(R.id.action_tutorial)
```

Comment:
```suggestion

                findItem(R.id.action_tutorial)?.isVisible = Locale.getDefault().language.startsWith("en")
```

Reply:
visibility can be set when creating the menu because the default language won't change without restarting the activity